### PR TITLE
Add scenario workflow for using Cert Manager to manage certificates

### PR DESCRIPTION
**Pull request by rohit2b**

### CODE REVIEW

File: security/using-cert-manager/README.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
+# Use Cert Manager to provide certificates for use
```
Comment (arodoni) [inline security/using-cert-manager/README.md:1]:
```suggestion
# Use Cert Manager to provide certificates
```

File: security/using-cert-manager/README.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+```
+kubectl -n confluent get zookeeper -oyaml
+kubectl -n confluent get schemaregistry -oyaml
+kubectl -n confluent get ksqldb -oyaml
+kubectl -n confluent get connect -oyaml
+kubectl -n confluent get controlcenter -oyaml
+```
+
+### ControlCenter UI
```
Comment (arodoni) [inline security/using-cert-manager/README.md:160]:
```suggestion
### Control Center UI
```

File: security/using-cert-manager/README.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+```
+kubectl apply -n confluent -f $TUTORIAL_HOME/resources/confluent-platform.yaml
+```
+
+## Validate
+
+### Status
+
+Use the status section to get information about the component: like endpoints, internal topics etc. Here, you'll see
```
Comment (arodoni) [inline security/using-cert-manager/README.md:145]:
```suggestion
Use the status section to get information about the component, such as endpoints, internal topics, etc. Here, you'll see 
```

**Review by arodoni — APPROVED** [review]
A few copy edits. Otherwise, LGTM.

**Review by xnulinu — APPROVED** [review]
LGTM @rohit2b thanks!

**Pull request merged**